src/main/java/shortestpath/ShortestPathPlugin.java

```diff
@@ -176,6 +206,31 @@ public boolean isNearPath(WorldPoint location) {
         return false;
     }
 
+    private final Pattern TRANSPORT_OPTIONS = Pattern.compile("avoidWilderness|useAgilityShortcuts|useGrappleShortcuts|useBoats|useCanoes|useCharterShips|useShips|useFairyRings|useGnomeGliders|useSpiritTrees|useTeleportationLevers|useTeleportationPortals");
```

Comment:
It is possible to grab all config keys with a particular prefix:
```java
@Inject
private ConfigManager configManager;
```
and then
```java
Pattern.compile("avoidWilderness|" + configManager.getConfigurationKeys(CONFIG_GROUP + ".use").stream().map(s -> s.substring(CONFIG_GROUP.length() + 1)).collect(Collectors.joining("|")));
```

Reply:
Since it's regex, I could simplify it to:
```java
private final Pattern TRANSPORT_OPTIONS_REGEX = Pattern.compile("^(avoidWilderness|use\\w+)$");
```
Functionally the same thing. Was intending to be more explicit with specifying keys but this also works.

Reply:
Feel free to change it to use regex. I just know that if I add another type of transport then I will forget to update the hardcoded string of configs